static/css/styles.css

```diff
@@ -0,0 +1,8 @@
+body {
+    margin-top: 20px;
+}
+
+.sidebar-nav {
+    padding: 0;
+    list-style: none;
+}
```

Comment:
no blank line at the end of file (

Reply:
![image](https://github.com/mate-academy/py-taxi-service-home-page/assets/84961728/b5c8d978-fec6-4391-b58c-195be9601e04)